.github/semantic.yml

```diff
@@ -0,0 +1,26 @@
+# Always validate the PR title AND all the commits
+titleAndCommits: true
```

Comment:
We haven't been using prefixes on the PR titles. Can we keep this just on the commits?

Reply:
We can, I just thought that if the commit is good then the PR automatically is. But not make things too difficult ok.